src/lib/history.ts

```diff
@@ -29,3 +26,22 @@ export const getArticleViewCountForWeeks = (
         0,
     );
 };
+
+export const historyWithinArticlesViewedSettings = (
```

Comment:
this code is copied from `src/lib/variants.ts ` so that it can be shared with banner tests